xformers/ops/memory_efficient_attention.py

```diff
@@ -810,11 +987,16 @@ def op(self) -> AttentionOp:
         """
         priority_list_ops: List[AttentionOp] = [
             MemoryEfficientAttentionFlashAttentionOp,
+            # TODO: remove once triton_faster_than_cutlass method complete
+            MemoryEfficientAttentionTritonFwdFlashBwOp,
             MemoryEfficientAttentionCutlassOp,
+            TritonFlashAttentionOp,
```

Comment:
I'm not sure we should include the Triton backward into the priority list, unless we are confident it works fine without correctness issues (Tri wasn't really sure about that).
Since it seems properly tested in the tests, I guess we should be good to go

Reply:
Makes sense, I removed it in #556 for now